recipe/conda_build_config.yaml

```diff
@@ -1,3 +1,2 @@
 cuda_version:
-  - "11"
   - "12+"
```

Comment:
We should be able to drop this entirely now and remove the whole `cuda_version` check. I'm happy to push the changes to your PR if desired.